<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8"/>
	<title>Guidelines</title>
	<meta name="generator" content="LibreOffice 5.1.4.2 (MacOSX)"/>
	<meta name="created" content="00:00:00"/>
	<meta name="changedby" content="Jocke Näsvall"/>
	<meta name="changed" content="2016-12-09T10:40:08.405159000"/>
	<meta name="Originator" content="Microsoft Word 14"/>
	<meta name="ProgId" content="Word.Document"/>
	<meta name="Title" content=""/>
	<meta name="created" content="00:00:00">
	<meta name="changedby" content="Jocke Nﾊsvall">
	<meta name="changed" content="2016-12-09T10:27:05.146429000">
	<meta name="created" content="00:00:00">
	<meta name="changedby" content="Jocke Nﾊsvall">
	<meta name="changed" content="2016-12-09T10:27:05.146429000">
	<meta name="Title" content="Strainlove Guidelines">
	</head>
<body lang="en-GB" text="#000000" link="#0000ff" vlink="#800080" bgcolor="#ffffff" dir="ltr">
<p style="margin-top: 0.42cm; page-break-after: avoid"><h1>Guidelines
for using the DA strain collection database (Dr. Strainlove)</h1></p>
<p style="margin-bottom: 0.5cm">Remember
that the descriptions of the strains in the strain collection should
be understandable by anybody in the lab, without access to your lab
notes. If you give a strain a temporary genotype while waiting for
sequencing results <i>etc.</i> remember to update it ASAP. <b>Please
do not enter a strain that does not yet exist into the database.</b>
<span style="font-weight: normal">If you really need to reserve
strain numbers before the strains are ready to be frozen, leave a
clear indication in the comment field that the strain is not frozen
yet.</span></p>

<p style="margin-bottom: 0.5cm"><h2>New users:</h2>
<p style="margin-bottom: 0.5cm"><b>Username:</b> Use something that you will remember.</p>
<p style="margin-bottom: 0.5cm"><b>Password:</b> Use something that you will remember.</p>
<p style="margin-bottom: 0.5cm"><b>Signature:</b> Needs to identify you as responsible for the strain and be understandable
by anybody, even after you have left the lab.
<p style="margin-bottom: 0.5cm"><b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Do not use just your initials!</b></p>
<p style="margin-bottom: 0.5cm"><b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Strong suggestion:</b> Your first name followed by additional initials.</p>
<p style="margin-bottom: 0.5cm"><b>User type:</b> "Superusers" are able to add and edit strains in the database and add new users. "Users" can only use the search functions.</p>

<p style="margin-bottom: 0.5cm"><h2>Genotype:</h2>
Should correctly identify the species, strain, isolate <i>etc</i>.
and contain a list of all mutations it has compared to the wild-type.
In cases when the mutation causes a defined phenotype, this may be
indicated in the description of the mutation <i>e.g.</i> <i>rpoB</i>(S531L,
Rif<sup>R</sup>), <i>etc</i>. <b>Evolved</b> strains should be indicated by “Evolved”
in the genotype.</p>
<p style="margin-bottom: 0.5cm">Please
keep other information (primer sequences, references, who you got the
strain from <i>etc</i>.) to the comment field. Avoid putting commas
or full stops in the genotype, and don’t put spaces in gene
designations. It makes the genotype open for misunderstanding.
<i>rpoB</i>(S531L) is correct, <i>rpoB</i> (S531L) is not. If you
notice a mistake in a genotype of your own strain, fix it. If you
notice a problem in the genotype of some other persons strain, ask
the person responsible for the strain to check it out and fix it if
necessary.</p>

<p style="margin-bottom: 0.2cm">Always start with an <b>indication of which species</b> the
strain is. Some examples of acceptable abbreviations and their
meaning:</p>
<table width="100%" cellpadding="4" cellspacing="0">
	<col width="14*">
	<col width="242*">
	<tr valign="top">
		<td width="6%" style="border: none; padding: 0cm">
			<p><b>Sty</b></p>
		</td>
		<td width="94%" style="border: none; padding: 0cm">
			<p><i>Salmonella</i>
			<i>enterica</i> subsp. <i>enterica</i> serovar Typhimurium (strain
			LT2 if not indicated otherwise)</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="6%" style="border: none; padding: 0cm">
			<p><b>Eco</b></p>
		</td>
		<td width="94%" style="border: none; padding: 0cm">
			<p><i>Escherichia
			coli</i> (K12 strain MG1655 if not indicated otherwise)</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="6%" style="border: none; padding: 0cm">
			<p><b>Kpn</b></p>
		</td>
		<td width="94%" style="border: none; padding: 0cm">
			<p><i>Klebsiella</i>
			<i>pneumonia</i></p>
		</td>
	</tr>
	<tr valign="top">
		<td width="6%" style="border: none; padding: 0cm">
			<p><b>Vna</b></p>
		</td>
		<td width="94%" style="border: none; padding: 0cm">
			<p><i>Vibrio
			natriegens</i></p>
		</td>
	</tr>
	<tr valign="top">
		<td width="6%" style="border: none; padding: 0cm">
			<p><b>Spy</b></p>
		</td>
		<td width="94%" style="border: none; padding: 0cm">
			<p><i>Streptococcus pyogenes</i></p>
		</td>
	</tr>
	<tr valign="top">
		<td width="6%" style="border: none; padding: 0cm">
			<p><b>Sau</b></p>
		</td>
		<td width="94%" style="border: none; padding: 0cm">
			<p><i>Staphylococcus aureus</i></p>
		</td>
	</tr>
</table>
<p style="margin-bottom: 0.5cm">

</p>
<p style="margin-bottom: 0.2cm">For
all other organisms than <i>E. coli</i> K12 MG1655 and <i>S.</i>
Typhimurium LT2, the species designation should always be followed by
an <b>indication of which strain</b> it is or which ATCC number it
has. For strains from other groups, put the original strain number in
the genotype, <i>e.g.</i>:</p>
<table width="100%" cellpadding="4" cellspacing="0">
	<col width="37*">
	<col width="219*">
	<tr valign="top">
		<td width="15%" style="border: none; padding: 0cm">
			<p><b>Sty
			14028</b> or <b>Sty ATCC14028</b></p>
		</td>
		<td width="85%" style="border: none; padding: 0cm">
			<p>Other
			strain than LT2</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="15%" style="border: none; padding: 0cm">
			<p><b>Eco
			K12 W3110 BW25113</b></p>
		</td>
		<td width="85%" style="border: none; padding: 0cm">
			<p>Strain
			from other group (BW=Barry Wanner)</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="15%" style="border: none; padding: 0cm">
			<p><b>Vna</b>
			<b>ATCC14048</b></p>
		</td>
		<td width="85%" style="border: none; padding: 0cm">
			<p>ATCC
			number</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="15%" style="border: none; padding: 0cm">
			<p><b>Sty
			TH194</b></p>
		</td>
		<td width="85%" style="border: none; padding: 0cm">
			<p>Strain
			from other group (TH= Diarmaid Hughes’ <i>Salmonella</i>
			collection)</p>
		</td>
	</tr>
</table>
<p style="margin-bottom: 0.5cm">

</p>

<p style="margin-bottom: 0.5cm">Use
as <b>correct
genetic nomenclature</b> as
possible. See <a href="http://www.sciencedirect.com/science/article/pii/S0076687906210012" target="_blank">Maloy
and Hughes (2007)</a> and <a href="http://www.genetics.org/content/54/1/61.long" target="_blank">Demerec
et al (1966)</a> for some detailed guidelines. Keep the same
designation for the same mutation in every strain it’s in. We don’t
assign allele numbers for mutations that we generate, but if a
mutation already has an allele number, keep it. When you have found
or constructed a mutant, make sure that the way you type it in is
unique in our database. If you <i>e.g.</i> construct
a new <i>hisA</i> deletion
that’s not identical to the one that’s already in the database
(DEL<i>hisA</i>),
you have to give a unique identifier to your mutant (<i>e.g.</i> DEL<i>hisA2</i>).</p>
<p style="margin-bottom: 0cm">

</p>
<p style="margin-bottom: 0.2cm"><b>Examples
for how to write specific mutations:</b></p>
<table width="100%" cellpadding="4" cellspacing="0">
	<col>
	<col>
	<tr valign="top">
		<td width="35%" style="border: none; padding: 0cm">
			<p>Deletion
			of one gene:</p>
		</td>
		<td width="65%" style="border: none; padding: 0cm">
			<p>DEL<i>hisA</i></p>
		</td>
	</tr>
	<tr valign="top">
		<td width="25%" style="border: none; padding: 0cm">
			<p>Deletion
			of a region containing more than one gene:</p>
		</td>
		<td width="75%" style="border: none; padding: 0cm">
			<p>DEL<i>araBAD</i>,
			DEL(<i>flhE-flhD</i>)</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="25%" style="border: none; padding: 0cm">
			<p>Insertion
			in a gene:</p>
		</td>
		<td width="75%" style="border: none; padding: 0cm">
			<p><i>miaB</i>::Tn<i>10</i>dTet</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="25%" style="border: none; padding: 0cm">
			<p style="font-variant: normal; font-style: normal">Replacement
			(= deletion plus insertion) of a gene:</p>
		</td>
		<td width="75%" style="border: none; padding: 0cm">
			<p>DEL<i>galE</i>::<i>Tlux</i><span style="font-variant: normal"><span style="font-style: normal">,
			DEL</span></span><i>hisA</i><span style="font-variant: normal"><span style="font-style: normal">::</span></span><i>kan</i></p>
		</td>
	</tr>
	<tr valign="top">
		<td width="25%" style="border: none; padding: 0cm">
			<p style="font-variant: normal; font-style: normal">Replacement
			of a region:</p>
		</td>
		<td width="75%" style="border: none; padding: 0cm">
			<p><span style="font-variant: normal"><span style="font-style: normal">DEL(</span></span><span style="display: inline-block; border: none; padding: 0cm"><span style="font-variant: normal"><span style="letter-spacing: normal"><i>hisOG</span></i></span></span><span style="font-variant: normal"><span style="letter-spacing: normal"><i><span style="font-weight: normal">DCBHAFI</span></i></span></span><span style="font-variant: normal"><span style="font-style: normal">)::</span></span><i>kan</i><span style="font-variant: normal"><span style="font-style: normal">,
			DEL(</span></span><i>hisO</i><span style="font-variant: normal"><span style="font-style: normal">-</span></span><i>hisI</i><span style="font-variant: normal"><span style="font-style: normal">)::</span></span><i>kan</i><span style="font-variant: normal"><span style="font-style: normal">,
			DEL(</span></span><i>araA</i><span style="font-variant: normal"><span style="font-style: normal">-</span></span><i>thiP</i><span style="font-variant: normal"><span style="font-style: normal">)::</span></span><i>cat</i></p>
		</td>
	</tr>
</table>
<p style="margin-bottom: 0.5cm">

</p>

<p style="margin-bottom: 0.2cm"><b>Rearrangements</b>
should ideally have an indication of where both junctions are:</p>
<table width="100%" cellpadding="4" cellspacing="0">
	<col width="38*">
	<col width="218*">
	<tr valign="top">
		<td width="35%" style="border: none; padding: 0cm">
			<p>Duplication:</p>
		</td>
		<td width="65%" style="border: none; padding: 0cm">
			<p>DUP(<i>rrnB</i>-<i>rrnE</i>)</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="15%" style="border: none; padding: 0cm">
			<p>Inversion:</p>
		</td>
		<td width="85%" style="border: none; padding: 0cm">
			<p>INV(<i>tufB</i>-<i>tufA</i>)</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="15%" style="border: none; padding: 0cm">
			<p>Rearrangement
			with a selection marker at the junction:</p>
		</td>
		<td width="85%" style="border: none; padding: 0cm">
			<p>DUP(<i>genE</i>)*<i>amp</i>*(<i>genX</i>)</p>
		</td>
	</tr>
</table>
<p style="margin-bottom: 0.5cm">

</p>

<p style="margin-bottom: 0.2cm"><h2>Parents:</h2></p>
<p style="margin-bottom: 0.5cm">The
<b>parental strain</b> is always written in the <b>recipient</b>
field. The <b>donor</b> field is restricted to the source of any DNA
that has been transferred into the parental strain by transformation,
transduction or conjugation. A strain that has been constructed by
direct selection for a spontaneous mutation or in an evolution
experiment thus only has a single parental strain (recipient), it has
no donor strain as no DNA has been transferred.</p>

<p style="margin-bottom: 0.2cm"><h2>Tips
for searches:</h2></p>
<p style="margin-bottom: 0.2cm"><b>Wildcards:</b></p>
<table width="100%" cellpadding="4" cellspacing="0">
	<col width="11*">
	<col width="245*">
	<tr valign="top">
		<td width="4%" style="border: none; padding: 0cm">
			<p>%</p>
		</td>
		<td width="96%" style="border: none; padding: 0cm">
			<p>Any
			string of characters. <b>his%</b> finds everything starting with
			his. <b>ara%kan</b> finds everything containing first <b>ara</b>
			and then <b>kan</b> separated by any number of characters.</p>
		</td>
	</tr>
	<tr valign="top">
		<td width="4%" style="border: none; padding: 0cm">
			<p>_</p>
		</td>
		<td width="96%" style="border: none; padding: 0cm">
			<p>Any
			single character. <b>his_::kan</b> finds everything starting with
			<b>his</b> separated by exactly one character from <b>::kan</b>
			(e.g. <i><b>hisA</b></i><b>::</b><i><b>kan</b></i><span style="font-weight: normal">,
			</span><i><b>hisF</b></i><b>::</b><i><b>kan</b></i> <span style="font-weight: normal">and
			</span><i><b>hisB</b></i><b>::</b><i><b>kan</b></i>). <b>hisA::_a_</b>
			will find e.g. <i><b>hisA</b></i><b>::</b><i><b>kan</b></i>,
			<i><b>hisA</b></i><b>::</b><i><b>cat</b></i> and <i><b>hisA</b></i><b>::</b><i><b>iAcatsac</b></i></p>
		</td>
	</tr>
</table>
<p style="margin-bottom: 0.5cm">

</p>
<p style="margin-bottom: 0cm">20161209
Jocke Näsvall</p>
<?php $helpmessage="<i>Salmonella</i> is the best genetic model organism!";?>
</body>
</html>
